plugins/dnswl.js

```diff
@@ -3,8 +3,8 @@
 
 exports.register = function() {
     var plugin = this;
-    plugin.load_dnswl_ini();
     plugin.inherits('dns_list_base');
+    plugin.load_dnswl_ini();
```

Comment:
It should be separate - I'm just dumb when it comes to git/github.

I meant to make it separate, but failed, and now I'm not sure how to fix it. =\